root/static/scripts/common/i18n/expand2react.js

```diff
@@ -19,6 +19,9 @@ import {
   l_admin as lAdminActual,
   ln_admin as lnAdminActual,
 } from '../i18n/admin.js';
+import {
+  l_history as lHistoryActual,
+} from '../i18n/history.js';
```

Comment:
Similarly to a previous comment, why following the model of the domain `statistics` instead of the model of the domain `attributes`?

Reply:
I think I needed this for `exp.l_history` in the `'{date} - {title}'` string - but if we have agreed to keep this as a `server` string instead, then it's not needed AFAICT. Will drop.